### DISCUSSION TITLE
Zephyrus g14 2021 power mode doesn't change

### DISCUSSION THREAD

**Discussion by DaliGaoudi** [seerge/g-helper#1197] [General]
Hi, i've been having a problem with switching power modes in the g-helper app. I had this bug a while ago and i just fixed it by restarting the laptop, i am having the same bug now and restarting it doesn't seem to fix it., i tried changing to older versions of ghelper and still nothing works. The GPU doesn't show up in the devices manager app and in the task manager performance tab. 
What do i do and do i resolve this issue for good because i don't want to go back to using armoury crate.
thanks

**Comment by seerge** [comment]
What do you mean under "power modes" ? And how is GPU connected to that ?

If your dGPU is not visible, you are probably in Eco mode, and you should switch to Standard in order to enable it.
If it's "stuck" by some reason - you can [try hardware reset](https://github.com/seerge/g-helper#how-do-i-do-a-hardware-reset-on-a-laptop)

**Reply by DaliGaoudi** [reply]
i mean by power modes: eco mode, standard and optimised. My problem is when i am in Eco mode and switch to Standard, the GPU doesn't turn on and when switching back to Eco mode, it shows  that it's changing but it doesn't and goes back to standard.

**Reply by seerge** [reply]
That's GPU modes, try hardware reset.

**Reply by DaliGaoudi** [reply]
all right thanks
